doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -168,6 +168,20 @@ Additional scenario XML elements:
   Specify whether force to disable software workaround for Machine Check
   Error on Page Size Change is enabled.
 
+``IVSHMEM`` (a child node of ``FEATURE``):
+  Specify the IVSHMEM configuration
+
+``IVSHMEM_ENABLED`` (a child node of ``FEATURE/IVSHMEM``):
+  Specify whether Enable Share Memory between VMs by IVSHMEM.
```

Comment:
I feel `shared` instead of `share` would read better throughout this document. @dbkinder , thoughts?

Reply:
ok